File: py_code/455_Distribution of biscuits.py
from typing import List

class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        g.sort()
        s.sort()
        i=0
        j=0
        while i<len(g) and j<len(s):
            if(s[j]>=g[i]):
                j+=1
                i+=1
            else:
                j+=1
        print(i)
        return i

s=Solution()
s.findContentChildren([1,2,3],[1,1])

File: js_code/正常顺序/455_Distribution of biscuits.js
/**
 * @param {number[]} g
 * @param {number[]} s
 * @return {number}
 */
 var findContentChildren = function(g, s) {
     g=g.sort((a,b) =>a-b);
     s=s.sort((a,b) =>a-b);
     let i=0,j=0,num=0;
     while(i<g.length&&j<s.length)
     {
       if(s[j]>=g[i])
       {
         i++;
         j++;
       }
       else
       {
         j++;
       }
     }
     console.log(i);
     return i;
};

findContentChildren([1,2,3],[1,1]);